src/_posts/databases/mysql/2000-01-01-managing.md

```diff
@@ -152,12 +161,11 @@ sure to follow the instructions on our [dedicated page]({% post_url databases/my
 
 ### Using the Database Dashboard
 
-1. From your web browser, open your [database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
-2. Click the **Overview** tab
-3. Locate the **Database Upgrade** block
-4. If an upgrade is available, the text in the block explains what will be
+1. From your web browser, [open your database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
+2. Click on the **Settings** tab
+3. In **General**, if an upgrade is available, the text in the block explains what will be
    done.
-5. To launch the upgrade, click the **Upgrade to ...** button
+4. To launch the upgrade, click the **Upgrade to ...** button
```

Comment:
Should we rather use the unicode character?

https://www.compart.com/en/unicode/U+2026

Reply:
Indeed, normally I've done it on all files